### ISSUE TITLE
✨Version differences

### ISSUE THREAD

**Issue by AntoLC** [numerique-gouv/impress#119]
## Feature Request

See: https://github.com/numerique-gouv/impress/issues/112

Could be nice to be able to see the difference between 2 versions like with `git diff` but directly on the editor.

The editor comes from [blocknote ](https://www.blocknotejs.org/). Blocknote is based on [ProseMirror](https://prosemirror.net/).
Maybe it exists a prosemirror plugin doing this feature, or talking about it.

**Comment by CelianGdfrd** [comment]
Could this idea be used to allow a multi-language wiki ? Like, switch from a French to an English version of a Docs page ?

**Comment by virgile-dev** [comment]
@CelianGdfrd I don't think so. But if you a multi lingual use case feel free to open a feature request and describe what you need.

**Comment by virgile-dev** [comment]
Thanks to Kevin Jahns from [Yjs](https://github.com/yjs/yjs) and his new [attribution feature](https://github.com/yjs/yjs/blob/4303ebdc9b1fc2fee2c43ced9f48d1b7d6e50420/attribution-manager.md) we'll soon be able to build this!

**Comment by lumoe** [comment]
Hi! Just a quick follow-up: is this feature still planned? Thanks!

**Comment by Ericson2314** [comment]
If the web app could actually sink the data to a git repo, and integrate that with "track changes", etc., I think that could be a killer feature that Google Docs, MS Office online, etc. don't have, and have seemingly no interest in adding.

**Comment by virgile-dev** [comment]
We're working on that with Yjs and BlockNote :)
